grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -593,12 +604,34 @@ class OrderService {
                         BigDecimal qtyPerUom = uomParts.length > 1 ? BigDecimal.valueOf(Double.valueOf(uomParts[1])) : null
                         orderItem.quantityUom = uom
                         orderItem.quantityPerUom = qtyPerUom
+                    } else {
+                        throw new IllegalArgumentException("Missing unit of measure.")
+                    }
+
+                    Integer parsedQty = Integer.valueOf(quantity)
+                    if (parsedQty <= 0) {
+                        throw new IllegalArgumentException("Wrong quantity value: ${parsedQty}.")
+                    }
+
+                    BigDecimal parsedUnitPrice = new BigDecimal(unitPrice).setScale(2, RoundingMode.FLOOR)
```

Comment:
We actually want this to be 4 places. We should ask Kelsey what she expects the rounding mode to be.

Reply:
Actually, sorry I'm probably wrong about the 4 places. That's only really needed for unit price on product.

Reply:
In the spreadsheet with the column description, Kelsey requested to limit to two decimal places when parsing values.